doc/README.md

```diff
@@ -1,12 +1,21 @@
 # Documentation for scikit-learn
 
-This section contains the full manual and web page as displayed in
+This directory contains the full manual and web site as displayed at
 http://scikit-learn.org.
-Building the website requires the sphinx and sphinx-gallery packages:
 
-    pip install sphinx sphinx-gallery
+## Building manually
 
-To generate the full web page, including the example gallery (this might take a
+Building the website requires the sphinx, sphinx-gallery and matplotlib
+packages:
+
+    pip install sphinx sphinx-gallery matplotlib
```

Comment:
Also `numpydoc` and `sphinx-gallery` (fixes https://github.com/scikit-learn/scikit-learn/issues/10064).

Reply:
Sorry, just the first one.